import-service/src/functions/import-file-parser/handler.ts

```diff
@@ -0,0 +1,66 @@
+import {
+  CopyObjectCommand,
+  DeleteObjectCommand,
+  GetObjectCommand,
+  S3Client,
+} from "@aws-sdk/client-s3";
+import { formatJSONErrorResponse, formatJSONResponse } from "@libs/api-gateway";
+import { middyfy } from "@libs/lambda";
+import { Handler, S3Event } from "aws-lambda";
+import { s3ClientConfig } from "src/config/s3-client-config";
+import { PARSED_FILES_FOLDER, UPLOADS_FOLDER } from "src/config/constants";
+
+const csv = require("csv-parser");
+
+const importFileParser: Handler<S3Event> = async ({ Records }) => {
```

Comment:
Highly suggested to log steps to know what exact step caused the issue. In AWS examples you may find that every `await` has its own `try..catch` block OR at least you have logs before every step.